podpac/core/settings.py

```diff
@@ -270,8 +272,9 @@ def load(self, path=None, filename='settings.json'):
         elif self['CACHE_DIR'] is None:
             self['CACHE_DIR'] = os.path.abspath(os.path.join(self['ROOT_PATH'], 'cache'))
 
-
-
-
 # load settings dict when module is loaded
 settings = PodpacSettings()
+
+if settings['LOG_TO_FILE']:
+    from podpac.core.utils import create_logfile
+    create_logfile()
```

Comment:
@mlshapiro This is something that will be very useful to me. I'd forget to set this up every time, but I'd like to have accurate logs of when opendap is down.